### ISSUE TITLE
Switch not showing after upgrade to Oruga 0.5.x

### ISSUE THREAD

**Issue by pflirae** [oruga-ui/theme-bulma#42]
After upgrading to Oruga 0.5, the Switch component did not show on screen, as can be seen here:
![image](https://user-images.githubusercontent.com/9519459/148994401-65e08d66-11b6-44da-b438-765f80967bbe.png)
if I remove theme-bulma, the switch appears again, as can be seen here:
![image](https://user-images.githubusercontent.com/9519459/148994685-6ab3765a-116f-4d6c-a7bc-ce27aad25932.png)
the HTML code is:

```
<OField label="Estado">
      <OSwitch v-model="estado_cliente"
                      true-value="Activo"
                      false-value="Inactivo"
        >
            {{ estado_cliente }}
         </OSwitch>
</OField>
```

and the app file is:

```
import './plageco_iconos'

import { createApp } from 'vue'
import App from './components/landing/App.vue'
import router from './router/index_back'
import store from './store'
import mitt from 'mitt'
import Oruga from '@oruga-ui/oruga-next'
import { bulmaConfig } from '@oruga-ui/theme-bulma'

import '../sass/backplageco.scss'

// import '@oruga-ui/oruga-next/dist/oruga.css'

import '@oruga-ui/theme-bulma/dist/bulma.css'

const plagecov1 = createApp(App)

const emitter = mitt()

plagecov1.config.globalProperties.emitter = emitter

// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
const customBulmaConfig = {
    ...bulmaConfig,
    iconPack: 'far'
}

// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
plagecov1.use(Oruga, customBulmaConfig)
// plagecov1.use(Oruga)

plagecov1.use(store).use(router).mount('#app')
```

I think that some detil sliped during the Oruga upgrade and is afecting theme-bulma.
Will be alert for any test request or aditional information needed.

**Comment by jtommy** [comment]
I'll release 0.2.0 soon, due to breaking changes there are some components to "fix".
Anyway the current master branch contains the correct code

**Comment by pflirae** [comment]
Ok, will close and wait for 0.2.x. Thanks

**Comment by jtommy** [comment]
@pflirae Just released 0.2.0 🎉
